exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -237,14 +239,14 @@ def _translate_to_json(self, spans: Sequence[Span]):
                     "otel.instrumentation_library.version"
                 ] = span.instrumentation_info.version
 
-            if span.status is not None:
-                zipkin_span["tags"]["otel.status_code"] = str(
-                    span.status.status_code.value
+            if span.status.status_code is not StatusCode.UNSET:
+                zipkin_span["tags"]["otel.status_code"] = status_code_to_str(
```

Comment:
Can't we simply use `statuscode.name`? Also, we would not want a dependency on `opentelemetry-instrumentation`.

Reply:
I didn't know `.name` thing, Thanks for that.

Reply:
It makes lot of sense why we would not want to have dependency on `opentelemetry-instrumentation`.